### DISCUSSION TITLE
`summon-workspace` different behavior with main/secondary monitors

### DISCUSSION THREAD

**Discussion by ok-nick** [nikitabobko/AeroSpace#737] [questions-and-answers]
Using `summon-workspace`, when I focus on the main monitor and the workspace I'm moving to is on an external monitor, it replaces the workspace on the main monitor. On the other hand, if I focus on an external monitor and select a workspace that's already on another monitor, it will move to that monitor, which is the behavior I'd expect everywhere. Is this behavior intentional?

**Comment by CheariX** [comment]
I think this is a bug.
Standard behavior should be to change the focus, I think.

There is also #681
